### ISSUE TITLE
earlyBrainDevelopment

### ISSUE THREAD

**Issue by ESapenaVentura** [ebi-ait/hca-ebi-wrangler-central#450]
### Project short name: 
earlyBrainDevelopment

### Primary Wrangler:
@ESapenaVentura 

### Secondary Wrangler:
@Wkt8 

### Associated files

* Google Drive: https://drive.google.com/drive/folders/17UGY7aXl9kBZ4G6Rtz_1aZX4BpsG14vy

### Published study links

* Paper: https://www.nature.com/articles/s41593-020-00794-1

* Accessioned data: https://assets.nemoarchive.org/dat-0rsydy7

### Key Events

- [x] Convert published metadata to HCA spreadsheet
- [x] Manually curate dataset to meet HCA metadata standard
- [x] Collect any matrix and cell-type annotation files
- [x] Upload sheet to validate metadata
- [x] Check linking using ingest graph validator
- [x] Transfer raw files to ingest to validate data files
- [x] Ask the Secondary Wrangler for an end-to-end review of the project. Ask the Expertise Wrangler to review specific tabs if needed
- [x] Submit dataset to Production 
- [x] Complete the Export SOP
- [x] Convert project data to SCEA format following the SCEA conversion SOP if appropriate

**Comment by Wkt8** [comment]
Lacking metadata on samples - might get through on today then needs secondary review.

**Comment by ESapenaVentura** [comment]
Project is here - https://contribute.data.humancellatlas.org/projects/detail?uuid=ef1d9888-fa86-47a4-bb72-0ab0f20f7004&tab=project

**Comment by Wkt8** [comment]
Found a few areas and had a chat with Enrique - will secondary review again when they are fixed!

**Comment by Wkt8** [comment]
Collection Protocol 
rather than the subdissection, I'd put that human fetal tissues were received from the  Human Developmental Biology Resource, staged using crown-rump length and shipped overnight on ice in Rosewell Park Memorial Institute media or in 4% paraformaldehyde.

Maybe add this url to the collection protocol as well
https://www.hdbr.org/uploads/default/factsheets/fixation_and_sectioning.pdf

Specimen from organism:
About half of the specimens don't have known disease 'normal' added to them 

In the Input Donor_Organism column you have 
CS14_3
CS15_2
CS22_2 
And they aren't in the donor organism tab 


Dissociation Protocol
Could move (Worthington, LK003150) into the Manufacturer and catalog number columns

**Comment by ESapenaVentura** [comment]
Many thanks for the review @Wkt8 !

I have added a new collection protocol (HDBR tissue collection), with the factsheet you mention uploaded as an attached supplementary file!

Good catch on the typos regarding IDs and disease, I have fixed them as well

Also moved the dissociation protocol's papaine manufacturer and catalog number to the proper fields.

I'm going to upload it to ingest and check linkings with the graph validator!
